Documentation/gettingstarted/host-firewall.rst

```diff
@@ -0,0 +1,156 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _host_firewall:
+
+********************
+Host Firewall (beta)
+********************
+
+This document serves as an introduction to Cilium's host firewall, to enforce
+security policies for Kubernetes nodes.
+
+.. include:: ../beta.rst
+
+Enable the Host Firewall in Cilium
+==================================
+
+.. include:: k8s-install-download-release.rst
+
+Deploy Cilium release via Helm:
+
+.. parsed-literal ::
+
+    helm install cilium |CHART_RELEASE|        \\
+      --namespace kube-system                  \\
+      --set global.kubeProxyReplacement=strict \\
+      --set global.hostFirewall=true           \\
+      --set global.devices=ethX,ethY
+
+At this point, the Cilium-managed nodes are ready to enforce network policies.
+
+
+Enable Policy Audit Mode for the Host Endpoint
+==============================================
+
+When applying `HostPolicies` carelessly, it's easy to block the access to the
+nodes or to break the cluster's normal behavior (ex. by blocking communication
+with kube-api).
```

Comment:
Maybe we can start with a more neutral start point than going straight to assuming careless policy apply behaviour ;-)

Here's a suggestion for alternative wording, feel free to take/modify for what you think makes sense:

```suggestion
`HostPolicies` enforce strict access control over connectivity to nodes. Particular care must be taken to ensure that when host policies are imported, Cilium does not block access to the nodes or break the cluster's normal behavior (for example by blocking communication with ``kube-apiserver``).
```

Reply:
Hehe, yep. I took your recommendation but dropped `strict` and extended `to nodes` to `to and from nodes`.